halo/attest/voter/voter.go

```diff
@@ -506,33 +513,35 @@ func (v *Voter) instrumentUnsafe() {
 	count(v.proposed, proposedCount)
 }
 
-// detectReorg returns an error if the previous block doesn't match the new block's parent hash.
-// This indicates that a reorg occurred.
-func detectReorg(chainVer xchain.ChainVersion, chainVerName string, prevBlock xchain.Block, block xchain.Block, cursors map[xchain.StreamID]uint64) error {
-	if prevBlock.BlockHash == (common.Hash{}) {
```

Comment:
this skipped omni consensus chain reorgs completely. We can still ensure height and message offsets are consecutive.